content/en/docs/ibm/deploy/install-kubeflow.md

```diff
@@ -75,21 +75,23 @@ get the best experience from Kubeflow.
 
 6. Set the Block Storage as the default storageclass.
     ```shell
-    kubectl patch storageclass ibmc-block-gold -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
+    NEW_STORAGE_CLASS=ibmc-block-gold
+    OLD_STORAGE_CLASS=$(kubectl get sc -o jsonpath='{.items[?(@.metadata.annotations.storageclass\.kubernetes\.io\/is-default-class=="true")].metadata.name}')
+    kubectl patch storageclass ${NEW_STORAGE_CLASS} -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
 
     # Check the default storageclass is block storage
-    kubectl get storageclass | grep \(default\)
+    kubectl get storageclass | grep "(default)"
     ```
 
     Example output:
     ```
     ibmc-block-gold (default)   ibm.io/ibmc-block   65s
     ```
 
-    Make sure `ibmc-block-gold` is the only `default` storageclass. If there are two or more rows in the above output, there is other `default` storageclass. Unset it with the below command, for example, will make the `ibmc-file-bronze` storage no longer the `default` storageclass for the cluster.
+    Make sure `ibmc-block-gold` is the only `default` storageclass. If there are two or more rows in the above output, there is other `default` storageclass. Unset it with the below command, for example, to remove the previous `default` storageclass for the cluster.
```

Comment:
Suggesting to replace "Unset it with ... to remove the previous `default` storageclass..." with "You can use the command below to remove the `default` storage class for the cluster:" (this eliminating some extra words - "unset... to remove" -> "remove")

WDYT?